DEVELOPMENT.md

```diff
@@ -529,3 +531,70 @@ $ git push origin master
 ```
 
 After pushing the tag, our CI system will start building the release. When this is done, you'll find a new release on the [releases page](https://github.com/TheThingsNetwork/lorawan-stack/releases). After this is done, you'll need to edit the release notes. We typically copy-paste these from `CHANGELOG.md`.
+
+
+### Console Troubleshooting
+
+#### Problem: Assets are not found
+The console will render a blank page and you can see backend logs like e.g.:
+```
+INFO Request handled                          duration=40.596µs error=error:pkg/errors/web:unknown (Not Found) message=Not Found method=GET namespace=web remote_addr=[::1]:50450 request_id=01DZ2CJDWKAFS10QD1NKZ1D56H status=404 url=/assets/console.36fcac90fa2408a19e4b.js
+```
+#### Possible causes
+##### Missing restart
+The stack has not been restarted after the console bundle has changed. In production mode, the stack will access the bundle via a filename that contains a content-hash, which is set during the build process of the console. The hash cannot be updated during runtime and will take effect only after a restart.
+
+##### Possible solution
+  1. Restart the stack
+
+##### Accidentally deleted bundle files
+The bundle files have been deleted. This might happen e.g. when a mage target encountered an error and quit before running through.
+
+##### Possible solution
+  1. Rebuild the console `mage js:clean js:build`
```

Comment:
Maybe having `./mage` is better here (to adhere with the rest of the document as well). Also in other places below